Contributors.md

```diff
@@ -519,6 +519,7 @@ aalbusoiu
 - [Tristan Marechaux] (https://github.com/marechaux)
 - [Deepak Sood](https://github.com/deepaksood619)
 - [Balamurugan](https://github.com/dillbala)
+  [Mrityunjay](https://https://github.com/Mj-Techs)
```

Comment:
You need to add a hyphen before the hyperlink, so that your name appears in a separate bullet point, see how Balamuragan did it. If you do it right your name will appear in a new line in the file preview.

See https://www.markdownguide.org/basic-syntax/#unordered-lists for an explanation of creating unordered lists in markdown.